chat-commands.js

```diff
@@ -70,6 +70,24 @@ exports.commands = {
 		"/auth [room] - Show what roomauth a room has.",
 		"/auth [user] - Show what global and roomauth a user has."],
 
+	userlist: function (target, room, user) {
+		let userList = [];
+
+		for (let i in room.users) {
+			let curUser = Users(room.users[i]);
+			if (!curUser || !curUser.named) continue;
+			userList.push(Chat.escapeHTML(curUser.name));
+		}
+
+		let output = "There " + Chat.plural(userList.length, 'are', 'is') + " <strong style=\"color:#24678d\">" + userList.length + "</strong> user" + Chat.plural(userList.length, 's', '') + " in this room:<br />";
```

Comment:
Also `Chat.plural(userList.length, 's', '')` is just `Chat.plural(userList.length)`